src/server/SocketServer/index.js

```diff
@@ -0,0 +1,49 @@
+const socketIo = require('socket.io');
+
+/**
+ * Abstract Api of socket.io lib
+ */
+class SocketServer {
+  /**
+   * Create a websocket whith an instant of http server
+   * @param {httpServer} httpServer
+   */
+  constructor(httpServer) {
+    this.io = socketIo(httpServer);
+  }
+
+  /**
+   * Call handler function when a new client is connected
+   * @param {Function} handler
+   */
+  connect(handler) {
+    this.io.on('connection', handler);
+  }
+
+  /**
+   * Call handler function when a client is disconnected
+   * @param {Function} handler
+   */
+  disconnect(handler) {
+    this.io.on('disconnect', handler);
+  }
+
+  /**
+   * Emit to all socket an event
+   * @param {string} event
+   * @param {object} data
+   */
+  emit(event, data) {
```

Comment:
prefer `ev` instead of `event`, in my mind, `event` is a nodejs module (and can be disturbing ? 🤔 )

Reply:
Okay i will update it